stories/components/Error.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react'
+import Error, { StatusCode } from '../../components/Error'
+import { MockedProvider } from '@apollo/client/testing'
+import GET_APP from '../../graphql/queries/getApp'
+
+export default {
+  components: Error,
+  title: 'Components/Error'
+}
+
+export const NotFound: React.FC = () => {
+  const mocks = [
+    {
+      request: { query: GET_APP },
+      result: {
+        data: {
+          lessons: [],
+          session: null,
+          alerts: []
+        }
+      }
+    }
+  ]
+  return (
+    <MockedProvider mocks={mocks} addTypename={false}>
+      <Error code={StatusCode.NOT_FOUND} message="404 error" />
+    </MockedProvider>
+  )
+}
+
+export const Internal: React.FC = () => {
+  const mocks = [
```

Comment:
I think this file can be refactored so that there is less duplicated code. For example, these `mocks` are exactly the same value in each storybook entry(`Internal, NotFound`)

Reply:
This should make it easier for the next person who has to update this file, since they don't have to make their own `mocks`, and they don't have to fill in `MockedProvider` with the correct prop values.

```
const mocks = [
  {
    request: { query: GET_APP },
    result: {
      data: {
        lessons: [],
        session: null,
        alerts: []
      }
    }
  }
]

const ErrorWrapper: React.FC = ({ children }) => (
  <MockedProvider mocks={mocks} addTypename={false}>
    {children as React.ReactElement}
  </MockedProvider>
)

export const NotFound: React.FC = () => (
  <ErrorWrapper>
    <Error code={StatusCode.NOT_FOUND} message="404 error" />
  </ErrorWrapper>
)

export const Internal: React.FC = () => (
  <ErrorWrapper>
    <Error code={StatusCode.INTERNAL_SERVER_ERROR} />
  </ErrorWrapper>
)
```

Reply:
Something similar to this:
```
const mocks = [
  {
    request: { query: GET_APP },
    result: {
      data: {
        lessons: [],
        session: null,
        alerts: []
      }
    }
  }
]

export const NotFound: React.FC = () => (
  <MockedProvider mocks={mocks} addTypename={false}>
    <Error code={StatusCode.NOT_FOUND} message="404 error" />
  </MockedProvider>
)

export const Internal: React.FC = () => (
  <MockedProvider mocks={mocks} addTypename={false}>
    <Error code={StatusCode.INTERNAL_SERVER_ERROR} />
  </MockedProvider>
)
```